### PR TITLE
docs: added section about local models via ollama

### PR DESCRIPTION
**Pull request by evilsocket**
ollama seems to be the most stable way of using local models, however it's not documented how to use it, so here it is

### CODE REVIEW

**Review by GangGreenTemperTatum — APPROVED** [review]

File: docs/topics/generators.md [hunk 1]
```diff
@@ -115,7 +115,27 @@ chats = await pipeline.run_many(50)
...
+ollama = rg.get_generator("ollama/llama3.1")
+```
+
+In the default configuration, the API base address is set to `http://localhost:11434`. It is possible to specify an alternative server by setting the OLLAMA_API_BASE environment variable:
+
+```py
+import os
+import rigging as rg
+
+os.environ['OLLAMA_API_BASE'] = 'http://192.168.0.10:11434'
```
Comment (monoxgas) [inline docs/topics/generators.md:133]: Would `,api_base=` on the generator identifier be a better alternative here that works out of the box? I typically don't like requiring env modifications for library functions. I can't remember how litellm does it, but we do pass that down into their methods.

File: docs/topics/generators.md [hunk 2]
```diff
@@ -115,7 +115,27 @@ chats = await pipeline.run_many(50)
...
+
+ollama = rg.get_generator("ollama/llama3.1")
```
Comment (monoxgas) [inline docs/topics/generators.md:135]: Can we clarify or add a note about whether ollama_chat is better here? https://docs.litellm.ai/docs/providers/ollama#using-ollama-apichat

File: docs/topics/generators.md [hunk 3]
```diff
@@ -115,7 +115,27 @@ chats = await pipeline.run_many(50)
 
 ## Local Models
 
-We have experimental support for both [`vLLM`](https://docs.vllm.ai/en/latest/) 
+Rigging supports local models via the [`ollama`](https://ollama.ai/) provider:
```
Comment (monoxgas) [inline docs/topics/generators.md:118]: Provided we essentially have 3 different "local providers" (given that ollama is technically just an API server as well), we might want to establish a short summary of the options and segment the technical details to their own sections.

**Comment by monoxgas** [comment]
Also, can you make this against dev instead?

File: docs/topics/generators.md (hunk 1 above)
Comment (evilsocket) [reply docs/topics/generators.md:133]: done, much better thanks!

File: docs/topics/generators.md (hunk 2 above)
Comment (evilsocket) [reply docs/topics/generators.md:135]: It's not super clear from their documentation, they only say:

> We recommend using [ollama_chat](https://docs.litellm.ai/docs/providers/ollama#using-ollama-apichat) for better responses.

and 

> In order to send ollama requests to POST /api/chat on your ollama server, set the model prefix to ollama_chat

Should I just add this? I can't tell from the docs in which exact way ollama_chat is better than just ollama.

File: docs/topics/generators.md (hunk 3 above)
Comment (evilsocket) [reply docs/topics/generators.md:118]: Just to make sure, we're talking LiteLLM, vLLM and Transformers right? The documentation mentions "experimental support" for vLLM, in which way is it less stable than the LiteLLM integration? I'll need some help about these sections as I'm not entirely sure of the support for each in Rigging and the main differences of using one vs another.

**Pull request closed without merging**